CONTRIBUTING.md

```diff
@@ -1,39 +1,167 @@
+![Commit Strip
+http://www.commitstrip.com/en/2014/05/07/the-truth-behind-open-source-apps/](docs/how-open-source-maintained.jpg)
+
+## Common issues and resolutions
+
+- Using `grape-active_model_serializers`, or any non-Rails server. See
+    [issue](https://github.com/rails-api/active_model_serializers/issues/1258).
+
 ## How can I help?
 
-Everyone is encouraged to open issues that are affecting you: bugs, ideas, performance problems – everything helps!
+### Filing an issue
+
+Everyone is encouraged to open issues that are affecting you:
+bugs, ideas, documentation, performance problems – everything helps!
+
+#### Before
+
+1. The first place to start is by looking at our [GitHub Issues](https://github.com/rails-api/active_model_serializers/issues).
+
+  - Check if your issue has already reported.
+  - If you find an existing issue report, feel free to add further information to that report.
+
+
+#### Writing
+
+If possible, please include the following information when [reporting an
+issue](https://github.com/rails-api/active_model_serializers/issues/new):
+
+- ActiveModelSerializers version (0.8.x, 0.9.x, 0.10.x, commit ref).
+- What are you using ActiveModelSerializers with? Rails? Grape? Other? Which versions?
+- If you are not running the latest version (please check), and you cannot update it,
+  please specify in your report why you can't update to the latest version.
+- Operating system type + version
+- Ruby version with patch level.  And if you're using rvm, rbenv, etc.
+  - Include your ruby -e "puts RUBY_DESCRIPTION".
+- Clearly-written steps to reproduce the issue (i.e. "Show me how to show myself." ), including:
+  - What were you doing? Include code if possible.
+    - Command line parameters used, if any.
+    - RubyGems code in your Gemfile, if any. Gemfile.lock, if possible
+    - Any configuration you've made
+  - What did you expect to happen?
+  - What happened? Include as much information as possible.
+    - Nature of reported defect (e.g. user name missing, not "It doesn't work."). Is it intermittent?
+    - The best help here is a failing test.
+    - Then the steps to reproduce and/or a gist or repository that demonstrates the defect.
+    - Then examples of the code you were using.
+    - Any error messages (including stacktrace, i.e. ""Show me the error.")
+  - Things you've tried.
+  - A pull request for your fix would be great.  Code should have tests.
+  - Link to source code, if available
+
+Please make sure only to include one issue per report.
+If you encounter multiple, unrelated issues, please report them as such.
+
+Simon Tatham has written an excellent on article on
+[How to Report Bugs Effectively](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html)
+which is well worth reading, although it is not specific to ActiveModelSerializers.
```

Comment:
Yeah, in response to the above I changed it in the README.  This gem suffers from naming confusion...